### PR TITLE
docs: add `test-gke` command to ci docs

### PR DESCRIPTION
**Pull request by nebril**

### CODE REVIEW

**Comment by nebril** [comment]
test-me-please

**Comment by nebril** [comment]
test-docs-please

File: Documentation/contributing/testing/ci.rst [hunk 1]
```diff
@@ -201,6 +201,8 @@ illustrating which subset of tests the job runs.
 +----------------------------------------------------------------------------------------------------------------+-------------------+--------------------+
 | `Cilium-PR-Flannel <https://jenkins.cilium.io/job/Cilium-PR-Flannel/>`_                                        | test-flannel      | No                 |
 +----------------------------------------------------------------------------------------------------------------+-------------------+--------------------+
+| `Cilium-PR-K8s-GKE <https://jenkins.cilium.io/job/Cilium-PR-K8s-GKE/>`_                                        | test-gke          | No                 |
```
Comment (joestringer) [inline Documentation/contributing/testing/ci.rst]: I guess this is Yes now :-)

```suggestion
| `Cilium-PR-K8s-GKE <https://jenkins.cilium.io/job/Cilium-PR-K8s-GKE/>`_                                        | test-gke          | Yes                 |
```

**Review by joestringer — CHANGES_REQUESTED** [review]
:tada:

**Review by joestringer — APPROVED** [review]

File: Documentation/contributing/testing/ci.rst [hunk 2]
```diff
@@ -201,6 +201,8 @@ illustrating which subset of tests the job runs.
 +----------------------------------------------------------------------------------------------------------------+-------------------+--------------------+
 | `Cilium-PR-Flannel <https://jenkins.cilium.io/job/Cilium-PR-Flannel/>`_                                        | test-flannel      | No                 |
 +----------------------------------------------------------------------------------------------------------------+-------------------+--------------------+
+| `Cilium-PR-K8s-GKE <https://jenkins.cilium.io/job/Cilium-PR-K8s-GKE/>`_                                        | test-gke          | Yes                |
```
Comment (qmonnet) [inline Documentation/contributing/testing/ci.rst:204]: Coming late to the party, but I think this test is now triggered with `test-me-please` as well, isn't it?

**Pull request merged**